pkg/wrapper/nmt_wrapper.go

```diff
@@ -25,19 +25,25 @@ type ErasuredNamespacedMerkleTree struct {
 	squareSize uint64 // note: this refers to the width of the original square before erasure-coded
 	options    []nmt.Option
 	tree       *nmt.NamespacedMerkleTree
-	idx        *rsmt2d.SquareIndex
+	// axisIndex is the index of the axis that this tree is on. This is passed
+	// by rsm2d and used to help determine which quadrant each leaf belongs to.
+	axisIndex uint64
+	// shareIndex is the index of the share that is being pushed to the tree. It
+	// is used to help determine which quadrant each leaf belongs to.
```

Comment:
[question] is this `shareIndex` the index of the share in the row or column that is specified by `axisIndex`? Is it worth disambiguating from the `shareIndex` we use elsewhere in this repo to describe the index in the square? If so, potential suggestion:

```suggestion
	// shareIndex is the index of the share in a row or column that is being
	// pushed to the tree. It is expected to be in the range: 0 <= shareIndex <
	// 2*squareSize. shareIndex is used to help determine which quadrant each
	// leaf belongs to.
```